### DISCUSSION TITLE
Grid Stat can't find Ensemble Stat output

### DISCUSSION THREAD

**Discussion by JeffHamiltonNOAA** [dtcenter/METplus#1110] [Statistical Computation] [answered]
Hello METplus team (@JohnHalleyGotway),

I'm attempting to use Grid Stat to verify reflectivity probabilistic output generated by Ensemble Stat, but am running into a problem after two thresholds have been read. The utility cannot find the additional probabilistic variables:

_WARNING: read_data_plane() -> REFC_L0_ENS_NMEP_ge25_NBRHD729_GUASSIAN1L0 not found in file: /home/amb-verif/ensemble/verif/cref/RRFSE/test/20210824-23z/ensemble_stat/ensemble_stat_011_G25_N40_20210824_230000V_ens.nc_

However, when plotting the same variable listed above myself within ncview, I get reasonable output that is in no way different in naming convention or scale than the two thresholds that were properly read (15 and 20 dBZ).

<img width="449" alt="Screen Shot 2021-08-25 at 7 20 53 AM" src="https://user-images.githubusercontent.com/43016574/130798079-77226c09-3876-4ab5-825c-b0889a64a5b3.png">

Is this behavior you have encountered before? Any attempts at tweaking and rerunning have failed thus far and I'm at an impasses since it doesn't make sense that a few thresholds would be found but not others when the variable naming convention is consistent throughout.

All of this work has been done on Jet, with the following log files revealing the issue:

workflow log file: /home/amb-verif/ensemble/verif/cref/RRFSE/log/RRFSE_cref_11_n40_grid_stat_2021082423.log
METplus log file: /home/amb-verif/ensemble/verif/cref/RRFSE/test/logs/master_metplus.log.20210825121027

Thanks in advance!

Jeff

**Comment by JohnHalleyGotway** [comment, marked as answer]
@JeffHamiltonNOAA, thanks for sending the path to the METplus log file. That makes this really easy to debug.
Please take a close look at your config file. You have typo with "GUASSIAN" spelled wrong. Please correct the spelling to "GAUSSIAN" and try again.

**Comment by JeffHamiltonNOAA** [comment]
*facepalm* well, it just goes to show how having another set of eyes can find a problem immediately. Thanks, John! It is working well now.
